src/main/java/stream/flarebot/flarebot/commands/music/SeekCommand.java

```diff
@@ -50,7 +45,7 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         return;
                     } else {
                         t.getTrack().setPosition(millis);
-                        MessageUtils.sendSuccessMessage("The track has been skipped to: " + GeneralUtils.formatJodaTime(period), channel);
+                        MessageUtils.sendSuccessMessage("The track has been skipped to: " + GeneralUtils.formatJodaTime(new Duration(millis).toPeriod()), channel);
```

Comment:
new Period(period)

Reply:
Nope, this is the only way given millis as a long, trust me I tried xD